Use new logo

pdhoopr opened the pull request:
Replacing the old logo with the new one (in green).

pdhoopr commented:
@mehdimas Whenever you get a chance, let me know what you think. I'll get it through to the actual site after that.

mehdimas commented:
Looks really cool. Can you change the menu underline colors to follow brand colors?

pdhoopr commented:
@mehdimas Okay, updated underline colors to be the green. I also updated the brand color, which changed the links - might want to verify those look alright to you, too.
